src/components/CodeEditor/CodeEditor.js

```diff
@@ -56,12 +56,10 @@ class CodeEditor extends Component {
     if (showBabelErrorMessage) {
       errorMessage = (
         <span>
-          Babel could not be loaded.
+          O Babel não pode ser carregado.
           <br />
           <br />
-          This can be caused by an ad blocker. If you're using one, consider
-          adding reactjs.org to the whitelist so the live code examples will
-          work.
+          Isto pode ser causado por algum bloqueador de anúncio. Se você está usando um, considere adicionar o endereço reactjs.org na <i>whitelist</i>, ou lista de permitidos, e feito isso os exemplos com <i>live code</i> irão funcionar.
```

Comment:
```suggestion
          Isto pode ser causado por algum bloqueador de anúncio. Se você está usando algum, considere adicionar o endereço reactjs.org na <i>whitelist</i>, ou lista de permitidos, e feito isso os exemplos com <i>live code</i> irão funcionar.
```

Reply:
I'm not sure whether we should use italics for non-Portuguese words or we should keep plain text.

cc @glaucia86

Reply:
https://github.com/reactjs/pt-BR.reactjs.org/issues/4#issuecomment-461955347